sql/moz-fx-data-shared-prod/relay_derived/subscriptions_v1/query.sql

```diff
@@ -0,0 +1,200 @@
+  FROM
+    `mozdata.subscription_platform.stripe_subscriptions_history`
+  WHERE
+    -- Only include the current history records and the last history records for previous plans.
+    (valid_to IS NULL OR plan_ended_at IS NOT NULL)
+    AND status NOT IN ("incomplete", "incomplete_expired")
+),
+relay_subscriptions AS (
+  SELECT
+    -- user_id,
+    customer_id,
+    subscription_sequence_id AS subscription_id,
+    IF(
+      subscription_sequence_id != subscription_id,
+      subscription_id,
+      NULL
+    ) AS original_subscription_id,
+    plan_id,
+    status,
+    synced_at AS event_timestamp,
+    IF(
+      (trial_end > TIMESTAMP(CURRENT_DATE) OR ended_at <= trial_end),
+      NULL,
+      COALESCE(plan_started_at, subscription_start_date)
+    ) AS subscription_start_date,
+    --first subscription start date associated with the subscription id
+    IF(
+      (trial_end > TIMESTAMP(CURRENT_DATE) OR ended_at <= trial_end),
+      NULL,
+      subscription_start_date
+    ) AS original_subscription_start_date,
+    IF(plan_started_at IS NOT NULL, "Plan Change", NULL) AS subscription_start_reason,
+    -- created,
+    trial_start,
+    trial_end,
+    canceled_at,
+    canceled_for_customer_at,
+    cancel_at,
+    cancel_at_period_end,
+    COALESCE(plan_ended_at, IF(ended_at < TIMESTAMP(CURRENT_DATE), ended_at, NULL)) AS ended_at,
+    IF(plan_ended_at IS NOT NULL, "Plan Change", NULL) AS ended_reason,
+    fxa_uid,
+    country,
+    country_name,
+    -- user_registration_date,
+    -- entrypoint_experiment,
+    -- entrypoint_variation,
+    -- utm_campaign,
+    -- utm_content,
+    -- utm_medium,
+    -- utm_source,
+    -- utm_term,
+    provider,
+    plan_amount,
+    -- billing_scheme,
+    plan_currency,
+    plan_interval,
+    plan_interval_count,
+    plan_interval_timezone,
+    product_id,
+    product_name,
+    CONCAT(
+      plan_interval_count,
+      "-",
+      plan_interval,
+      "-",
+      plan_currency,
+      "-",
+      (plan_amount / 100)
+    ) AS pricing_plan,
+    -- Stripe billing grace period is 7 day and Paypal is billed by Stripe
+    INTERVAL 7 DAY AS billing_grace_period,
+    promotion_codes,
+    promotion_discounts_amount,
+  FROM
+    stripe_subscriptions_history
+  LEFT JOIN
+    standardized_country
+  USING
+    (country)
+  -- LEFT JOIN
+  --   users
+  -- USING
+  --   (fxa_uid)
+  -- LEFT JOIN
+  --   attribution
+  -- USING
+  --   (fxa_uid)
+  WHERE
+    (
+      "premium-relay" IN UNNEST(stripe_subscriptions_history.product_capabilities)
+      OR "premium-relay" IN UNNEST(stripe_subscriptions_history.plan_capabilities)
+    )
+),
+relay_subscriptions_with_end_date AS (
+  SELECT
+    *,
+    IF(
+      customer_id IS NOT NULL,
+      MIN(subscription_start_date) OVER (PARTITION BY customer_id),
+      subscription_start_date
+    ) AS customer_start_date,
+    COALESCE(ended_at, TIMESTAMP(CURRENT_DATE)) AS end_date,
+  FROM
+    stripe_subscriptions
```

Comment:
The CTE name has changed.
```suggestion
    relay_subscriptions
```